File: Iniciante/1097 - Sequencia IJ 3/JavaScript/1097.js
function main() {
  let initialValueJ = 7;

  for (let i = 1; i <= 9; i += 2) {
    for (let j = initialValueJ; j >= initialValueJ - 2; j--) {
      console.log(`I=${i} J=${j}`);
    }
    initialValueJ += 2;
  }
}

main();

File: Iniciante/1097 - Sequencia IJ 3/Python/1097.py
def main():
    initial_value_j = 7

    for i in range(1, 10, 2):
        for j in range(initial_value_j, initial_value_j - 3, -1):
            print(f"I={i} J={j}")
        initial_value_j += 2


if __name__ == "__main__":
    main()
